src/fsharp/FSharp.Core/list.fsi

```diff
@@ -156,6 +302,18 @@ namespace Microsoft.FSharp.Collections
         /// <param name="list">The input list.</param>
         ///
         /// <returns>The result list.</returns>
+        /// 
+        /// <example id="countBy-example-1">
+        /// <code lang="fsharp">
+        /// let isEven x = 0 = x % 2
```

Comment:
This example feels a bit too mathematical (modulus) - better to be getting a statistic of some kind, e.g. count a list of lists by length, or strings by number of occurrences of characters

Reply:
Counted the letters in the `['H'; 'a'; 'p'; 'p'; 'i'; 'n'; 'e'; 's'; 's']` . Do you think that will be better to use `"Happiness".ToCharArray() |> Array.toList` ?